cpp/kiss_icp/metrics/Metrics.cpp

```diff
@@ -148,8 +148,8 @@ std::tuple<float, float> SeqError(const std::vector<Eigen::Matrix4d> &poses_gt,
         r_err += it.r_err;
     }
 
-    double avg_trans_error = avg_trans_error = 100.0 * (t_err / float(err.size()));
-    double avg_rot_error = avg_rot_error = 100.0 * (r_err / float(err.size())) / 3.14 * 180.0;
+    double avg_trans_error = 100.0 * (t_err / static_cast<double>(err.size()));
+    double avg_rot_error = 100.0 * (r_err / static_cast<double>(err.size())) / 3.14 * 180.0;
```

Comment:
The changes to the cast of `err.size()` weren't strictly necessary, but 1) casting to `float` when everything else is `double` didn't seem like the right thing and 2) doing `float(...)` or `double(...)` is essentially a C-style cast, which in C++ should be avoided.

Reply:
True, this code was mainly not touched as was copy-psted from the kitti-dev benchmark code. Maybe some metrics will change, but we are not submitting to the benchmark again, so it looks like a safe move ;)